pkg/addons/device_plugin.go

```diff
@@ -21,6 +21,7 @@ const (
 	// Currently only us-east-1 and us-west-2 are available
 	// and both use this AWS account
 	neuronResourceAccount = "790709498068"
+	efaResourceAccount    = "TODO"
```

Comment:
👀

Reply:
encase I have to do this, where would you find this value :smile: ?

Reply:
@michaelbeaumont encase I have to do this, where would you find this value smile ?

Reply:
added the account information